src/components/dialogs/parameters/dynamicsimulation/curve/dialog/equipment-filter.jsx

```diff
@@ -427,10 +416,11 @@ const EquipmentFilter = forwardRef(
                         <CheckboxAutocomplete
                             id="nominal-voltage"
                             options={nominalVoltages}
+                            value={selectedNominalVoltages}
                             getOptionLabel={(value) =>
                                 `${value} ${NOMINAL_VOLTAGE_UNIT}`
                             }
-                            onChangeCallback={handleNominalVoltageChange}
+                            onChange={handleNominalVoltageChange}
```

Comment:
same

Reply:
DONE in https://github.com/gridsuite/gridstudy-app/pull/1836/commits/0bef9a2e0bd4c319f0e8f31aa7ea0cd139c91dc1

Reply:
No, just put directly the setter

Reply:
DONE